zendev/cmd/serviced.py

```diff
@@ -320,6 +320,11 @@ def run_serviced(args, env):
         if wait_for_ready:
             print "serviced is ready!"
 
+        # opt_serviced/var/isvcs needs 755 perms
+        var_isvcs = environ.servicedhome.join('var', 'isvcs').__str__()
+        if subprocess.call(["sudo", "chmod", "755", var_isvcs]):
```

Comment:
Does this have to be done on startup, or can we just refine the cleanup on/around line 39?